code/parachain/frame/fnft/src/lib.rs

```diff
@@ -199,6 +208,25 @@ pub mod pallet {
 		OptionQuery,
 	>;
 
+	#[pallet::call]
+	impl<T: Config> Pallet<T> {
+		/// transfer fnft to a new owner
+		#[pallet::weight(T::WeightInfo::transfer())]
+		pub fn transfer(
+			origin: OriginFor<T>,
+			collection: FinancialNftCollectionIdOf<T>,
+			instance: FinancialNftInstanceIdOf<T>,
+			destination: AccountIdOf<T>,
+		) -> DispatchResult {
+			let who = ensure_signed(origin)?;
+			let fnfts = OwnerInstances::<T>::get(&who).ok_or(Error::<T>::MustBeOwner)?;
+			ensure!(fnfts.contains(&(collection, instance)), Error::<T>::MustBeOwner);
```

Comment:
```suggestion
			ensure!(fnfts.contains(&(collection, instance)), Error::<T>::InstanceNotFound);
```

Reply:
should i change the error to InstanceNotFound to the line above as well then? There is MustBeOwner Error  defined in pallet but it is not used anywhere.

Reply:
`MustBeOwner` makes sense for the previous line no?

Reply:
Correct, I read it wrong.